brainstormforce/ultimate-addons-for-gutenberg
# Fix:Post Parent-child category editor issues

[Trello Task](https://trello.com/c/V1UT94nB/136-uag-posts-block-to-have-a-selection-for-related-posts)
### Description
Resolve parent-child category posts issue in the editor.

### Types of changes
- Added function to check the parent-child category relationship in the taxonomy list to display posts in different layouts (Grid, Masonry, Carousel).
- Removed Select option in taxonomyListOptions.

### Test Cases
- Create parent-child categories.
- Add categories to the posts.
- In the Editor Please select any parent category.
- Check It will display all the related posts.

### How has this been tested?
Tested whether it is displaying all the related posts in the editor.

### Checklist:
